generated/nirfmxbluetooth/nirfmxbluetooth.proto

```diff
@@ -623,8 +648,7 @@ message ACPFetchMeasurementStatusRequest {
 
 message ACPFetchMeasurementStatusResponse {
   int32 status = 1;
-  AcpResultsMeasurementStatus measurement_status = 2;
-  int32 measurement_status_raw = 3;
+  int32 measurement_status = 2;
```

Comment:
Commented in [the scrapigen PR](https://github.com/ni/grpc-device-scrapigen/pull/204) so hopefully we can avoid this binary-breaking change.